.github/workflows/quickstart.yml

```diff
@@ -85,4 +85,4 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: clippy
-          args: -- -D warnings
\ No newline at end of file
+          args: -- -D warnings -A clippy::upper-case-acronyms
```

Comment:
please put a newline at the end of the file

Reply:
It's a fair cop guvnor. Fixed.